python/Ganga/Core/GangaRepository/SubJobXMLList.py

```diff
@@ -407,10 +413,13 @@ def getAllCachedData(self):
         #logger.debug( "Cache: %s" % str(self._subjobIndexData.keys()) )
         if len(self._subjobIndexData.keys()) == len(self):
             for i in range(len(self)):
```

Comment:
Probably. But I don't know 100% for certain that all that enumerate calls is len here as I don't want to iterate over the class at all in this step for a few minor reasons. I'll do some investigating in the future but as it stands this PR isn't responsible for this so I'm going to do this elsewhere.